### DISCUSSION TITLE
Голосовые звонки и новый взгяд на связь

### DISCUSSION THREAD

**Discussion by PetrBronzo** [octothorp-team/public-dobrograd#784] [Предложения]
Предложение: Добавить голосовые звонки по телефону и стационарному телефону, что добавит реалистичности в РП процесс при условии того что в телефоне будет создан по цифрам свой номер и будет передаваться как частота рации, данная возможность реализуется по полной в рекламе, РП процессе, обычных ситуациях, требующих разговора между игроками

Да! Это улучшит мое РП и я буду общаться с друзьями без пищащей Рации
Нет! Я люблю общаться через рацию

**Comment by outclown** [comment]
Есть куча тем таких, я создал за несколько часов до твоей тему про телефоны, в которой так же это расписано

**Reply by PetrBronzo** [reply]
Увидел только после создания поста
